modules/local/gfa_to_fasta.nf

```diff
@@ -0,0 +1,28 @@
+process GFA_TO_FASTA {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda (params.enable_conda ? "conda-forge::gawk=5.1.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gawk:5.1.0' :
+        'quay.io/biocontainers/gawk:5.1.0' }"
+
+    input:
+    tuple val(meta), path(gfa)
+
+    output:
+    tuple val(meta), path("*.fa"),         emit: fasta
+    path "versions.yml",                   emit: versions
+
+    script:
+    def args = task.ext.args ?: ''
+    """
+    prefix=\$(basename $gfa .gfa)
```

Comment:
Why is this `prefix` inside the code instead of outside like for the other modules?